defaults/main.yml

```diff
@@ -100,6 +100,9 @@ datadog_ignore_old_centos_python3_error: false
 # Use the datadog_zypper_repo variable to override the repository used.
 datadog_zypper_repo: ""
 
+# Define if the official zypper Datadog repository services should be installed
+datadog_zypper_repo_enabled: yes
```

Comment:
Could you rename this to `datadog_manage_zypper_repo`? The `enabled` keyword is a repofile keyword and it might the variable name confusing. Additionally, we already have a variable called `datadog_manage_config`, so using `datadog_manage_zypper_repo` would be consistent with that.

Reply:
Actually, even better might be `datadog_manage_zypper_repofile`.

Reply:
Hi, I changed the variable name according to your suggestion.
Thanks.